lib/index.js

```diff
@@ -0,0 +1,143 @@
+async function goHome() {
+  await browser.goTo('#Shell-home');
+  await browser.waitForUI5();
```

Comment:
where are `goTo` and `waitForUI5()` injected into the browser object?
a comment here might help the inclined developer :)
plus: if this is primarily targeting `wdi5`, the `wdi5.goTo()` seems more appropriate.
plus: we should refrain from any manual `waitForXX` APIs and rather leave it to the test framework to sync app lifecyle with test lifecycle. E.g. with `wdi5`, no waiting for any ops to finish is necessary as the underlying RecordReplay-API takes care of this and is accounted for in `wdi5`

Reply:
switched to wdi5.goTo()